Extract service

nouhouari opened the pull request:
Add Appointment service that will be used  by API and Controller.

alextselegidis commented:
Hello! 

Thanks for the PR!

Unfortunately the proposed solution is not sufficiently organized and does not match the application code style. 

I was also having different implementation in mind for this, so that I will need to close this PR. 

The required changes for the offending item will be pushed later today on `develop` branch. 

Stay tuned!

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="https://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

nouhouari commented:
Hi @alextselegidis,
I've just refactored the code and moved some existing functions to the AppointmentService.
Therefore, I didn't change the code style.
Is the issue related to the coding conventions or the architecture change (library)?
Could you help clarifying the code style that is expected for me to contribute ?

Note: I've pushed more changes to refactor the Google Calendar Synch and Email notification.

Regards
